tools/debian/convert_debian.py

```diff
@@ -34,97 +36,179 @@
 # e.g. [buster] - xz-utils 5.2.4-1+deb10u1
 VERSION_PATTERN = re.compile(r'\[(.*?)\]\s*-\s*([^\s]+)\s*([^\s]+)')
 
+# e.g. <define-tag moreinfo>\n Some html here \n</define-tag>
+WML_DESCRIPTION_PATTERN = re.compile(r'<define-tag moreinfo>((?:.|\n)*)</define-tag>', re.MULTILINE)
```

Comment:
Is there a more robust/reliable way to parse these? e.g. using an xml parser?

Reply:
thought of using an XML parser, but the default python one doesn't seem to like the format of the `.wml` and `.data` files. 
I'll look into this more, maybe it's a configuration I need to set.

Reply:
Ack. Can you add a TODO here?

Reply:
Did some more testing here, turns out the `.data` format is not valid XML, since all attributes in XML has to have a value associated with it (at least according to this answer: https://stackoverflow.com/a/6928130). 
 
I tried a html parser instead and it sort of works, though required wrapping the file with a root element. I'm not sure if that's much more reliable compared to the regex version. 

I have just added a TODO there for now.

Reply:
Ack, regex may be OK in this case then. Thanks -- TODO sounds good!